static/style.css

```diff
@@ -106,30 +106,44 @@ button:focus {
     margin-bottom: 40px;
 }
 
+
+
```

Comment:
No, those were a mistake I didn't notice until I commited. I'm still quite a newbie at this Git thing, so  I'm not sure how to fix that after I commited.